src/js/controller/settings/exportimage/ExportController.js

```diff
@@ -22,7 +22,6 @@
 
   ns.ExportController = function (piskelController, i18n) {
     this.createExportDom_(i18n);
-
```

Comment:
accidental deletion?

Reply:
Did I remove a space to separate the lines of code?